doc/source/releases/22.01_announce.rst

```diff
@@ -0,0 +1,114 @@
+
+===========================================================
+January 2022 Galaxy Release (v 22.01)
+===========================================================
+
+.. include:: _header.rst
+
+Highlights
+===========================================================
+
+**Galaxy starts as fastAPI application by default**
+  Starting galaxy via ``run.sh`` will use the new `gravity process manager
+  <https://github.com/galaxyproject/gravity>`__.
+  The new configuration uses `gunicorn <https://gunicorn.org/>`__ and `fastAPI <https://fastapi.tiangolo.com/>`__ to drive Galaxy's web process and starts job handler and celery processes.
+  For more details see the documentation at <WRITE AND PUT DOCS HERE>.
+  `Pull Request 13224`_
+
+**User Preferences can be encrypted in Galaxy Vault**
+  @nuwang, could you write something about this please?
```

Comment:
@nuwang do you have a moment to write a short summary here ?

Reply:
Just saw this. Will do.

Reply:
Done.

Reply:
Thank you!